### PR TITLE
feat: editor package simpler language strings

### PR DESCRIPTION
**Pull request by hejtful**
Changes to match https://github.com/serlo/frontend/pull/3496:
- No need to explicitly import `instanceDataDe` and `loggedInDataDe`
- Language data instead provided by `SerloEditor` component
- Overwriting language strings therefore moved to the `children` prop of `SerloEditor` component (still considered a hack, but no regressions from the previous version

In order to test locally, please follow the instructions added to the README.

Before merging, `@serlo/editor` package needs to be published and its version updated here.

### CODE REVIEW

File: src/frontend/editor.tsx [hunk 1]
```diff
@@ -37,19 +34,14 @@ export interface EditorProps {
...
   return (
-    <SerloEditorPackage
-      initialState={state.document}
-      instanceData={instanceDataDe as SerloEditorProps['instanceData']}
-      loggedInData={loggedInDataDe as SerloEditorProps['loggedInData']}
-    >
-      {(editor) => {
+    <SerloEditorPackage initialState={state.document}>
+      {(editor, languageData) => {
+        // HACK: Change strings in link element. Searching or inserting an id is not possible in this integration.
```
Comment (elbotho) [inline src/frontend/editor.tsx:40]: Not even sure if I would consider this a hack. I mean we need some way of overwriting default strings, but also don't encourage it…

**Review by elbotho — APPROVED** [review]
also thanks for working out and documenting the workflow, looks clean and convenient 👍

File: src/frontend/editor.tsx (hunk 1 above)
Comment (hejtful) [reply src/frontend/editor.tsx:40]: Agreed, just thought that Lars had some reason to mark it as a hack, so I didn't want to remove the comment without consulting him.

Comment (LarsTheGlidingSquirrel) [reply src/frontend/editor.tsx:40]: I think the readability would improve if we got rid of the 'hack'. I had a quick look into the code and in `link-controls.tsx` we have boolean `isSerloLinkSearchActive` which could control what strings are displayed internally. I'll make the change if there is no veto.

**Review by LarsTheGlidingSquirrel — APPROVED** [review]
Looks cleaner now! Nice, that you also documented how to use the editor package locally with `yalc`.

**Pull request merged**